tier0/{{cookiecutter.project_slug}}/repolinter.json

```diff
@@ -0,0 +1,177 @@
+{
+    "extends": "https://raw.githubusercontent.com/DSACMS/repo-scaffolder/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/repolinter.json",
```

Comment:
Since Tier 0 is now considered the tier with our baseline repository requirements, I think it may be worth refactoring our repolinter.json files such that Tier 0 is the parent and Tier 1-4 extends from this